### PR TITLE
vc based exporter

### PR DESCRIPTION
**Pull request by richardtief**
The main idea is, to query the inventory with the right target. That means, starting multiple exporters with different targets and the exporters will fetch the related resources with the given target from inventory.

### CODE REVIEW

File: collectors/ClusterPropertiesCollector.py [hunk 1]
```diff
@@ -94,3 +75,12 @@ def do_metrics(self, target, gauges, infos, states):
                     labels=[self.clusters[cluster_id]['name'],
                             self.clusters[cluster_id]['parent_dc_name'].lower()],
                     value={metric_suffix: info_value})
+
+        # self.post_metrics(self.g.name)
+        # self.post_metrics(self.i.name + '_info')
```
Comment (viennaa) [inline collectors/ClusterPropertiesCollector.py:80]: We can remove this now

File: exporter.py [hunk 2]
```diff
@@ -22,8 +22,10 @@ def parse_params():
                       dest="collectors")
     parser.add_option("-m", "--config", help="path to config to set default collectors, statkeys and properties for "
                                              "collectors", action="store", dest="config")
+    parser.add_option("-t", "--target", help="define target vcenter", action="store", dest="target")
```
Comment (viennaa) [inline exporter.py:25]: The way I got it this should be the target vrops, shouldn't it?

**Review by viennaa — CHANGES_REQUESTED** [review]
Very solid implementation. Two minor things and we're good to go from my side.

File: collectors/ClusterPropertiesCollector.py (hunk 1 above)
Comment (richardtief) [reply collectors/ClusterPropertiesCollector.py:80]: With this, I removed all the `CollectorUp` Code.

File: exporter.py (hunk 2 above)
Comment (richardtief) [reply exporter.py:25]: Your right. This have to be `vrops` instead of `vcenter`.

File: tools/vrops.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import os
 
 
-class Resources:
+class Vrops:
```
Comment (viennaa) [inline tools/vrops.py:10]: Can we rename the file starting with uppercase `V`?

Comment (richardtief) [reply tools/vrops.py]: Indeed.

File: tests/TestCollectorInit.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 
 class TestCollectorInitialization(TestCase):
-    os.environ['TARGET'] = 'testhost.test'
+    os.environ.setdefault('TARGET', 'testhost.test')
```
Comment (viennaa) [inline tests/TestCollectorInit.py:15]: 👍  ;)

**Review by viennaa — APPROVED** [review]
Thanks, let's go!

**Pull request merged**